src/test/resources/features/list.feature

```diff
@@ -1,29 +1,33 @@
 Feature: List actions
 
-  @ArchiveList
+  @CreateList
   Scenario: New list creation
     Given I log in as admin user
-    When I select TestBoard board
+    When I select BoardForList board
     And I create a new list with "ListTest" as title
     Then I should see the new created list with the given title
 
   @MoveAllCards
   Scenario: Move all cards in list
     Given I am logged in as admin user
-    When I select TestBoard board
-    And I move all cards in Tasks1 to Tasks2
+    And I select MoveSortBoard board
+    And there are following lists in the board
+      | SourceList |
+      | TargetList |
+    When I move all cards in SourceList to TargetList
     Then all cards that were on source list should appear on target list
     And the source list should be empty
 
   @SortCardsByName
   Scenario: Sort cards in list by card name
     Given I am logged in as admin user
-    When I select TestBoard board
-    And I sort cards in Tasks1 list by card name
+    And I select MoveSortBoard board
```

Comment:
Justify this line please.

Reply:
It is necessary to select one board to make the tests. In this case, MoveSortBoard was created by an API before hook.